playbooks/openshift-glusterfs/private/add_hosts.yml

```diff
@@ -0,0 +1,12 @@
+---
+# This play runs when new gluster hosts are part of new_nodes group during
+# master or node scaleup.
+- import_playbook: gluster_hosts.yml
+  vars:
+    l_glusterfs_hosts: "glusterfs:&oo_nodes_to_config"
```

Comment:
Just to make sure, I imagine in the context of a scaleup `oo_nodes_to_config` only contains the nodes to be added?